runtimes/api.py

```
@@ -109,11 +144,44 @@ async def update_policy(
+    if body.pools is not None:
+        rich["pools"] = body.pools
+    if body.policy is not None:
+        rich["policy"] = body.policy
+    if body.triggers is not None:
+        rich["triggers"] = body.triggers
+    if rich:
+        existing = await _load_rich_policy()
+        existing.update(rich)
+        try:
+            await _save_rich_policy(existing)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)